test/metadata-stream_test.js

```diff
@@ -233,7 +233,7 @@
 
     equal(events.length, 1, 'parsed one tag');
     equal(events[0].frames.length, 1, 'parsed one frame');
-    equal(events[0].frames[0].id, 'WXXX', 'parsed the frame id');
```

Comment:
would be nice if all the descriptions were updated to match the new property too